docs/api/experimental.rst

```diff
@@ -133,6 +133,9 @@ Attributes
 
     * ``beginObject`` and ``eindeObject`` are made read-only. `The reason <https://github.com/VNG-Realisatie/gemma-zaken/issues/2332>`__
 
+    * ``betrokkeneIdentificatie.identificatie`` max length is changed from 24 to 128  for ``betrokkeneType: "medewerker"``
```

Comment:
Could you move this to the `Rol` resource on this page? It can be listed below line 43 of this file